.github/scripts/check_newsfragments.py

```diff
@@ -50,19 +55,12 @@ def check_newsfragment(fragment: Path, base: str) -> Optional[bool]:
             print(
                 f"[{fragment_name}] fragment ID doesn't correspond to an issue ! (On <{url}> HTTP error {exc.code} {exc.reason})"
             )
+            return False
         else:
+            # Sanity check is we can deserialize the response.
```

Comment:
The sanity check IS the deserialization? If so, I'd rephrase it like this:
```suggestion
            # Sanity check: try to deserialize the response.
```